tensorflow/examples/label_image/BUILD

```diff
@@ -12,12 +12,32 @@ cc_binary(
     srcs = [
         "main.cc",
     ],
-    linkopts = ["-lm"],
-    deps = [
-        "//tensorflow/cc:cc_ops",
-        "//tensorflow/core:framework_internal",
-        "//tensorflow/core:tensorflow",
-    ],
+    linkopts = select({
+        "//tensorflow:android": [
+            "-pie",
+            "-landroid",
+            "-ljnigraphics",
+            "-llog",
+            "-lm",
+            "-z defs",
+            "-s",
+            "-Wl,--exclude-libs,ALL",
+        ],
+        "//conditions:default": ["-lm",],
+    }),
+    deps = select({
+        "//tensorflow:android": [
+            "//tensorflow/cc:cc_ops",
```

Comment:
Nope, gif/jpeg/png dependencies are not linked in, because decode_{gif/jpeg/png}_op are not included in android_tensorflow_lib.

I'll add a comment.